### PR TITLE
Sms 4444 sdk 10dlc enhancement

### PR DESCRIPTION
**Pull request by NirmitiJain**

### CODE REVIEW

**Review by prashantjha-plivo — CHANGES_REQUESTED** [review]

File: src/main/java/com/plivo/api/models/brand/BrandUsecaseResponse.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BrandUsecaseResponse extends BaseResource {
+
+    private List<useCase> useCases = new ArrayList<useCase>();
+    private String brandId;
+
+    /**
+     * @return String return the getBrandId
+     */
+    public String getBrandId() {
```
[the remark this answers is not in the document]
Comment (NirmitiJain) [reply src/main/java/com/plivo/api/models/brand/BrandUsecaseResponse.java:20]: As per requirement, we need to return field called "brand_id"
and extending BaseResource mandates overriding getId()

File: src/main/java/com/plivo/api/PlivoAPIService.java [hunk 2]
```diff
@@ -256,6 +258,9 @@ Call<ListResponse<Brand>> brandList(@Path("authId") String authId,
   @GET("Account/{authId}/10dlc/Brand/{id}/")
   Call<Brand> brandGet(@Path("authId") String authId, @Path("id") String brandId);
 
+  @GET("Account/{authId}/10dlc/Brand/{id}/usecases/")
```
Comment (narayana-plivo) [inline src/main/java/com/plivo/api/PlivoAPIService.java:261]: nit : usecases => Usecases

File: src/main/java/com/plivo/api/models/brand/BrandUsecase.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import com.plivo.api.models.base.BaseResource;
+
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BrandUsecase extends BaseResource {  
+  private List<useCase> useCases = new ArrayList<useCase>();
```
Comment (narayana-plivo) [inline src/main/java/com/plivo/api/models/brand/BrandUsecase.java:13]: nit: useCase => UseCase

File: src/main/java/com/plivo/api/models/brand/useCase.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package com.plivo.api.models.brand;
+
+
+public class useCase {
```
Comment (narayana-plivo) [inline src/main/java/com/plivo/api/models/brand/useCase.java:4]: class name should be capital letter

File: src/main/java/com/plivo/api/models/brand/useCase.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    private String name;
+    private String code;
+    private String details;
```
Comment (narayana-plivo) [inline src/main/java/com/plivo/api/models/brand/useCase.java:7]: details or detail?

**Review by narayana-plivo — CHANGES_REQUESTED** [review]
@NirmitiJain please address the request change and update the versions

File: src/main/java/com/plivo/api/models/brand/useCase.java (hunk 5 above)
Comment (NirmitiJain) [reply src/main/java/com/plivo/api/models/brand/useCase.java:7]: It's "details" as per PRD

File: README.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ If you are using Maven, use the following XML to include the Plivo SDK as a depe
 <dependency>
   <groupId>com.plivo</groupId>
   <artifactId>plivo-java</artifactId>
-  <version>5.13.0</version>
+  <version>6.13.0</version>
```
Comment (narayana-plivo) [inline README.md:21]: please update 5.14.0

**Review by narayana-plivo — APPROVED** [review]
LGTM. Good job @NirmitiJain

**Comment by narayana-plivo** [comment]
@NirmitiJain Build and UT is failing. Please check

**Review by narayana-plivo — CHANGES_REQUESTED** [review]
Please fix UT and Build

**Review by prashantjha-plivo — APPROVED** [review]
LGTM

**Pull request merged**